### DISCUSSION TITLE
V2Ray(VLESS+H2C)+Caddy中漏掉了host配置导致无法连接

### DISCUSSION THREAD

**Discussion by lochuan** [lxhao61/integrated-examples#107] [Announcements]
https://github.com/lxhao61/integrated-examples/blob/main/V2Ray(VLESS%2BH2C)%2BCaddy/1_v2ray_config.json

httpSettings里面漏掉了host，查了好久才发现

**Comment by lxhao61** [comment]
本示例没有错，httpSettings里面不需要配置host。
你自己有错，客户端HTTP/2 Host为空即可！

**Comment by lochuan** [comment]
不是客户端，是服务器上的xray，测试了很多次，发现不加host 的话无法处理caddy 转发过来的h2c的流量

**Comment by sjtuross** [comment]
遇到了一样的问题，小火箭host留空连接不上，passwall里留空则可以，但我看官方h2示例是要有host的

**Comment by lxhao61** [comment]
> 遇到了一样的问题，小火箭host留空连接不上，passwall里留空则可以，但我看官方h2示例是要有host的

小火箭问题（其它客户端测试都正常）。官方 H2C/H2 示例是我很早之前提交的，当时延续 Caddy v1 版保留了 host 配置，且站点直接 xx.yy 会自动加匹配 host，而 443, xx.yy 不会。
